internal/cloudfront/origin.go

```diff
@@ -33,11 +43,15 @@ type Origin struct {
 	Behaviors []Behavior
 	// ResponseTimeout is how long CloudFront will wait for a response from the Origin in seconds
 	ResponseTimeout int64
+	// Type is this Origin's type (s3 bucket, public, etc)
+	Type string
```

Comment:
Maybe make more sense to be `Access` instead of `Type`. wdyt?

Reply:
This was actually added as part of https://github.com/gympass/cdn-origin-controller/pull/83 😅, will be removed when we rebase this one. I'll add the comment there.

Reply:
I'm sorry hehehe